account_financial_risk/models/res_partner.py

```diff
@@ -215,11 +215,15 @@ def _compute_risk_exception(self):
                 max_value = getattr(partner, risk_field[1], 0.0)
                 if max_value and field_value > max_value:
```

Comment:
Please do it more Pythonic :wink: :

```suggestion
                partner.risk_exception = max_value and field_value > max_value
```